auth-server-open-api-spec.yaml

```diff
@@ -797,6 +796,25 @@ components:
           description: A description of the rights associated with this access token.
           items:
             $ref: '#/components/schemas/access'
+          allOf:
+            - contains:
+                properties:
+                  type:
+                    const: incoming-payment
```

Comment:
I tried to find the type `const` in the OpenAPI3.1 spec but I wasn't able to. Where did you find this?

Reply:
It's included in the version of JSON Schema used in OpenAPI 3.1
https://json-schema.org/understanding-json-schema/reference/generic.html#constant-values